openassessment/xblock/submission_mixin.py

```diff
@@ -170,7 +170,17 @@ def submit(self, data, suffix=''):  # pylint: disable=unused-argument
                     for answer_err in err.field_errors.get('answer', [])
                 )
                 if answer_too_long:
+                    logger.exception(
+                        f"Attempted to submit response for {student_item_dict} but the submission is too long"
+                    )
```

Comment:
Nit: putting the student item dict at the end of the log message would improve search-ability and proximity to the surfaced message would help as well.

```suggestion
                    logger.exception(
                        f"Response exceeds maximum allowed size: {student_item_dict}"
                    )
```

Reply:
I was about to get sarcastically annoyed because at firt I thought I'd have to re-do the translations nonsense but then I realized this is a log message and not translated! wahoo